ApplicationMapping with a list of services (#4114)
kyma-project/kyma: docs/application-connector/06-02-apllicationmapping.md

## Sample custom resource

> This is a sample ApplicationMapping resource which enables the `test` Application in the `production` Namespace (all services provided by the Application are enabled):

Reviewer: ```suggestion
This is a sample ApplicationMapping resource which enables the `test` Application in the `production` Namespace. In this example, all services provided by the Application are enabled:
```

```yaml
apiVersion: applicationconnector.kyma-project.io/v1alpha1
kind: ApplicationMapping
metadata:
  name: test
  namespace: production
```

> The ApplicationMapping could enable only selected services in the given Namespace, see the example:

Reviewer: ```suggestion
Using ApplicationMapping, you can also enable only the selected services in a given Namespace. See the example:
```

```yaml
apiVersion: applicationconnector.kyma-project.io/v1alpha1
kind: ApplicationMapping
metadata:
  name: test
  namespace: production
spec:
  services:
    - id: ac031e8c-9aa4-4cb7-8999-0d358726ffaa
    - id: bef3143c-d1a5-674c-8dc9-ab4788896fba
```

> The `services` list contains items with a service ID, which must be enabled. The `service` can be null, which means all services of the Application are enabled. If the list is specified - only such services are enabled.

Reviewer: ```suggestion
The `services` list contains IDs of enabled services. If the list is specified, only the selected services are enabled. If the list is empty, all services of the Application are enabled.
```

> The `services` list contains items with a service ID, which must be enabled. The `service` can be null, which means all services of the Application are enabled. If the list is specified - only such services are enabled.

Reviewer 2: ```suggestion
The `services` list contains items with a service ID, which are enabled. The `service` can be null, that means all services of the Application are enabled. If the list is specified - only such services are enabled.
```
